sonar-php-plugin/src/test/java/org/sonar/plugins/php/reports/phpstan/PhpStanSensorTest.java

```diff
@@ -183,4 +193,22 @@ protected ExternalIssuesSensor sensor() {
   protected LogTester logTester() {
     return logTester;
   }
+
+  private static String osIndependentPath(String path){
+    if (SEPARATOR_CHAR == '\\'){
+      path =  path.replace("/", "\\");
+    }
+    return path;
+  }
+
+  private static String reportPath(String path){
+    if (SEPARATOR_CHAR == '\\'){
+      path =  path.replace("/", "\\");
+      StringBuilder builder = new StringBuilder(path);
+      int index = builder.lastIndexOf(".");
+      builder.replace(index, index+1, "_win.");
+      return builder.toString();
+    }
+    return path;
+  }
 }
```

Comment:
I can see this method exist in both `PsalmSensorTest.java` and `PhpStanSensorTest.java`, maybe we can consider moving it in parent class `ReportSensorTest` + with the SEPARATOR_CHAR protected attribute, if that make sens to you.
Also I can see that the test `SEPARATOR_CHAR == '\\'` is used several time in both classes, we may also consider creating a protected method `isWindowsSeparator()` instead.
WDYT ?